openmetadata-ui/src/main/resources/ui/src/utils/CustomProperties/CustomProperty.utils.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { ExtentionEntities } from 'components/common/CustomPropertyTable/CustomPropertyTable.interface';
+import { EntityType, TabSpecificField } from 'enums/entity.enum';
+import { getDashboardByFqn } from 'rest/dashboardAPI';
+import {
+  getDatabaseDetailsByFQN,
+  getDatabaseSchemaDetailsByFQN,
+} from 'rest/databaseAPI';
+import { getGlossaryTermByFQN } from 'rest/glossaryAPI';
+import { getMlModelByFQN } from 'rest/mlModelAPI';
+import { getPipelineByFqn } from 'rest/pipelineAPI';
+import { getSearchIndexDetailsByFQN } from 'rest/SearchIndexAPI';
+import { getContainerByFQN } from 'rest/storageAPI';
+import { getStoredProceduresDetailsByFQN } from 'rest/storedProceduresAPI';
+import { getTableDetailsByFQN } from 'rest/tableAPI';
+import { getTopicByFqn } from 'rest/topicsAPI';
+
+export const getEntityExtentionDetailsFromEntityType = <
```

Comment:
Can we utilise same method which we have used under AssetsUtils `getAPIfromSource`.